etl/cris_import/README.md

```diff
@@ -2,21 +2,23 @@
 
 This ETL manages the processing and importing of TxDOT CRIS data into the Vision Zero database.
 
-All data processing is managed by a single script, `cris_import.py` which processes both CSV files and CR3 PDF crash reports. The script supports a number of CLI arguments to handle a number of data processing scenarios.
+The following scripts are available:
 
-## Quick start
+- **`cris_import.py`**: the primary data import script which processes both CSV files and CR3 PDF crash reports.
+- **`cr3_ocr_narrative.py`**: a utility script which extracts crash narrative data from CR3 PDFs using Optical Character Recognition (OCR)
+- **`_restore_zips_from_archive.py`**: a development helper script that moves CRIS extract zips from `./archive` to `./inbox` in the S3 bucket.
 
-Set your `ENV` to `dev` in order to safely run the S3 operations locally.
+## Quick start - CRIS import
 
 1. Start your local Vision Zero cluster (database + Hasura + editor).
 
-2. Save a copy of the `env_template` file as `.env`, and fill in the details.
+2. Save a copy of the `env_template` file as `.env`, and fill in the details. Set the `BUCKET_ENV` variable to `dev` in order to safely run the S3 operations locally.
```

Comment:
i decided to change the env var `ENV` to `BUCKET_ENV`, just to be super clear that this is the only S3 operations are affected by this variable. For example, you could set `BUCKET_ENV` to `prod` and still point your hasura endpoint/secret to local, and vice-versa.

Reply:
I like this change, what do you think about updating the env_tamplate with the new var name?